### ISSUE TITLE
Why use as Conv2D use_bias=not batch_normalize in Convert Model 

### ISSUE THREAD

**Issue by zhukkang** [qqwweee/keras-yolo3#288]
in   convert.py line 168 , why as batch_normalize on will set use_bias to False ?
   conv_layer = (Conv2D(
                filters, (size, size),
                strides=(stride, stride),
                kernel_regularizer=l2(weight_decay),
                use_bias=not batch_normalize,
                weights=conv_weights,
                activation=act_fn,
                padding=padding))(prev_layer)

**Comment by dzhus** [comment]
BatchNorm layers will shift activations by their mean values, so there's no point in adding any constants anyways – setting `use_bias=False` saves time and memory here.

**Comment by leoluopy** [comment]
> BatchNorm layers will shift activations by their mean values, so there's no point in adding any constants anyways – setting `use_bias=False` saves time and memory here.

impressive answer
